lookout/style/format/langs/javascript/uast_fixers.py

```diff
@@ -12,6 +12,31 @@ def fix_regexp_node(node: bblfsh.Node) -> bblfsh.Node:
     return node
 
 
+def fix_string_literal_type_anotation(node: bblfsh.Node) -> bblfsh.Node:
```

Comment:
According to https://github.com/bblfsh/javascript-driver/issues/37 `fix_regexp_node` should be fixed in 2.7.1, so we can remove it. Let's check

Reply:
that's true.

Reply:
But you mean `fix_regexp_node` get it.